automation/src/test/scala/GoogleSpec.scala

```diff
@@ -0,0 +1,26 @@
+import org.broadinstitute.dsde.firecloud.test.WebBrowserSpec
+import org.scalatest.ParallelTestExecution
+import org.scalatest.selenium.WebBrowser
+import org.scalatest.FlatSpec
+
+/**
+  * Example test for running with Selenium Grid until we work out docker
+  * networking to be able to access FiaBitC.
+  */
+class GoogleSpec extends FlatSpec with WebBrowserSpec with ParallelTestExecution with WebBrowser {
+
+  behavior of "Google"
+
+  it should "have a search field" in withWebDriver { implicit driver =>
+    go to new GooglePage()
+    assert(find("q").isDefined)
```

Comment:
I think you mentioned this in email, but I generally prefer ScalaTest matchers over `assert(boolean)` because it gives nicer error messages. If you mix in the `ShouldMatchers` trait this could be written as:
```
find("q") shouldBe 'defined
```